src/stitching/delegateToSchema.ts

```diff
@@ -91,14 +93,27 @@ async function delegateToSchemaImplementation(
   }
 
   if (options.operation === 'subscription') {
-    // apply result processing ???
-    return subscribe(
+    const executionResult = await subscribe(
       options.schema,
       processedRequest.document,
       info.rootValue,
       options.context,
       processedRequest.variables,
-    );
+    ) as AsyncIterator<ExecutionResult>;
+
+    // "subscribe" to the subscription result and map the result through the transforms
+    return mapAsyncIterator<ExecutionResult, any>(executionResult, (result) => {
+      const transformedResult = applyResultTransforms(result, transforms);
+      const subscriptionKey = Object.keys(result.data)[0];
+
+      // for some reason the returned transformedResult needs to be nested inside the root subscription field
```

Comment:
Hm. That's weird. Can you check 'CheckResultAndHandleErrors'? It's supposed to handle this I think.

Reply:
Ah, never mind. I think it's just the way subscription works in this case.